torch/optim/adadelta.py

```diff
@@ -394,16 +395,10 @@ def _multi_tensor_adadelta(
             torch._foreach_add_(device_state_steps, 1)
 
         if maximize:
-            device_grads = torch._foreach_neg(device_grads)
+            torch._foreach_neg_(device_grads)
```

Comment:
this would inplace modify the grads which is not okay

Reply:
@david20571015 let's just type ignore

Reply:
ok